odoo/addons/base/views/res_company_views.xml

```diff
@@ -57,14 +67,28 @@
             <field name="name">res.company.tree</field>
             <field name="model">res.company</field>
             <field name="arch" type="xml">
-                <tree string="Companies">
+                <tree string="Companies" decoration-muted="not active">
                     <field name="sequence" widget="handle"/>
                     <field name="name"/>
                     <field name="partner_id"/>
                     <field name="child_ids" widget="many2many_tags" options="{'color_field': 'color'}"/>
+                    <field name="active" invisible="1"/>
                 </tree>
             </field>
         </record>
+        <record id="view_company_branches_tree" model="ir.ui.view">
+            <field name="name">res.company.branches.tree</field>
+            <field name="model">res.company</field>
+            <field name="inherit_id" ref="base.view_company_tree"/>
+            <field name="mode">primary</field>
+            <field name="priority">100</field>
+            <field name="arch" type="xml">
+                <field name="active" position="attributes">
+                    <attribute name="widget">boolean_toggle</attribute>
+                    <attribute name="invisible">0</attribute>
```

Comment:
Note for the forward-port: `invisible`'s value must be Python code, so on master this will become
```suggestion
                    <attribute name="invisible">False</attribute>
```

Reply:
Isn't `0` a valid python expression?

Reply:
To be consistent, I changed both this one and the `invisible="1"` higher up to their boolean counterparts.